flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/util/HBaseConfigurationUtil.java

```diff
@@ -22,20 +22,85 @@
 import org.apache.flink.util.Preconditions;
 
 import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HBaseConfiguration;
 import org.apache.hadoop.io.Writable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.DataInputStream;
 import java.io.DataOutputStream;
+import java.io.File;
 import java.io.IOException;
 
 /**
- * This class helps to do serialization for hadoop Configuration.
+ * This class helps to do serialization for hadoop Configuration and HBase-related classes.
  */
 @Internal
 public class HBaseConfigurationUtil {
 
+	private static final Logger LOG = LoggerFactory.getLogger(HBaseConfigurationUtil.class);
+
+	@SuppressWarnings("deprecation")
+	public static Configuration getHBaseConfiguration() {
+
+		// Instantiate an HBaseConfiguration to load the hbase-default.xml and hbase-site.xml from the classpath.
+		Configuration result = new HBaseConfiguration().create();
```

Comment:
```suggestion
		Configuration result = HBaseConfiguration.create();
```

Reply:
yep, new HBaseConfiguration() is marked deprecation.

Reply:
It seems this line is not updated in the new commit, maybe you missed the change? @liuyongvs  Thanks.

Reply:
ok,  I mistook that it would add automatically when triggering the  button "Commit changes".